Retos/Reto #4 - PRIMO, FIBONACCI Y PAR [Media]/javascript/EmaPaul.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Escribe un programa que, dado un número, compruebe y muestre si es primo, fibonacci y par.
+ * Ejemplos:
+ * - Con el número 2, nos dirá: "2 es primo, fibonacci y es par"
+ * - Con el número 7, nos dirá: "7 es primo, no es fibonacci y es impar"
+*/
+
+function esPrimo(n){
+    
+    if(n===2){
+        return `${n} es primo, `
+    }else if(n<2){
+        return `${n} no es primo, `
+    }else if(n==0){
+        return `${n} no es primo, `
+    }for(let i = 2; i<n; i++){
+        if(n%i===0){
+            return `${n} no es primo, `
+        }
+    }
+    
+    return `${n} es primo, `
```

Comment:
Hola!! Tengo algunos consejos para que tu código quede más limpio. Por ejemplo: puedes evitar usar los `else` ya que no son necesarios en este caso. Además puedes omitir la validación de la línea 14, `if(n==0)` porque el caso de la línea 12 `if (n < 2)` ya contiene la validación de ese escenario. Si tienes alguna no dudes en escribirme o ponerlo acá mismo en el thread 😄  Saludos!!

```suggestion
    if(n===2){
        return `${n} es primo, `
    }
    if(n<2){
        return `${n} no es primo, `
    }
    for(let i = 2; i<n; i++){
        if(n%i===0){
            return `${n} no es primo, `
        }
    }
    
    return `${n} es primo, `
```

Reply:
Grande @EmaPaul ! vi el cambio ya realizado, excelente, que bueno que te pudo ayudar el comentario 😄 Lo único es que creo que no tengo permiso para dar por "Resuelto" el thread, si a ti te deja, dale nomás!
Un saludo!

Reply:
no me deja tendré que esperar unas horas creo ....  : (